packages/clay-management-toolbar/src/ActiveState.tsx

```diff
@@ -0,0 +1,248 @@
+								? labels.checkbox
+								: undefined
+						}
+						onChange={onCheckboxChange}
+					/>
+				</Item>
+				<Item>
+					<span className="navbar-text">
+						{selectedItems === totalItems ? (
+							'all-selected'
+						) : (
+							<>
+								{`${selectedItems}-of-${totalItems} `}
+								<span className="navbar-breakpoint-down-d-none">
+									{'selected'}
+								</span>
+							</>
+						)}
+					</span>
+				</Item>
+
+				{supportsBulkActions && (
+					<Item className="nav-item-shrink">
+						{clearSelectionURL ? (
+							<ClayLink
+								className="nav-link"
+								href={clearSelectionURL}
+							>
+								<span className="text-truncate-inline">
+									<span className="text-truncate">
+										{labels.clearSelection}
+									</span>
+								</span>
+							</ClayLink>
+						) : (
+							<ClayButton
+								className="nav-link"
+								displayType="unstyled"
+								onClick={onClearSelectionClick}
+								type="button"
+							>
+								<span className="text-truncate-inline">
+									<span className="text-truncate">
+										{labels.clearSelection}
+									</span>
+								</span>
+							</ClayButton>
+						)}
+					</Item>
+				)}
+
+				{showSelectAllButton && (
+					<Item className="nav-item-shrink">
+						{selectAllURL ? (
+							<ClayLink className="nav-link" href={selectAllURL}>
+								<span className="text-truncate-inline">
+									<span className="text-truncate">
+										{labels.selectAll}
+									</span>
+								</span>
+							</ClayLink>
+						) : (
+							<ClayButton
+								className="nav-link"
+								displayType="unstyled"
+								onClick={onSelectAllClick}
+								type="button"
+							>
+								<span className="text-truncate-inline">
+									<span className="text-truncate">
+										{labels.selectAll}
+									</span>
+								</span>
+							</ClayButton>
+						)}
+					</Item>
+				)}
+			</ItemList>
+
+			{(actionItems.length > 0 || showInfoButton) && (
+				<>
+					<ItemList>
+						{showInfoButton && (
+							<Item>
+								<ClayButton
+									className="nav-link nav-link-monospaced"
+									displayType="unstyled"
+									onClick={onInfoButtonClick}
+									title={titles.infoButton}
+								>
+									<ClayIcon
+										spritemap={spritemap}
+										symbol="info-circle-open"
+									/>
+								</ClayButton>
+							</Item>
+						)}
+					</ItemList>
+
+					{actionItems.length > 0 && (
+						<>
+							{actionItems.map(
+								({quickAction, ...item}, index) =>
+									quickAction && (
+										<Item
+											className="navbar-breakpoint-down-d-none"
+											key={index}
+										>
+											{item.href && !item.disabled ? (
+												<ClayLink
+													className="nav-link nav-link-monospaced"
+													href={item.href}
+													title={item.label}
+												>
+													<ClayIcon
+														spritemap={spritemap}
+														symbol={
+															item.symbolLeft as string
```

Comment:
How come this doesnt get properly inferred?

Reply:
This is due to the API that `item.symbolLeft` is optional but I think I can check for `symbolLeft` before using it here. Maybe here https://github.com/liferay/clay/pull/2225/files/502893529d4160abe9d2faee596ee4c89e270126#diff-ceb43593f9c76576fc542f9f9c9f3c7bR182